src/corelibs/U2Core/src/datatype/msa/MultipleAlignment.cpp

```diff
@@ -55,10 +55,11 @@ const MultipleAlignmentData *MultipleAlignment::operator->() const {
     return maData.data();
 }
 
-MultipleAlignmentData::MultipleAlignmentData(const QString &name, const DNAAlphabet *alphabet, const QList<MultipleAlignmentRow> &rows)
-    : alphabet(alphabet),
-      rows(rows),
-      length(0) {
+MultipleAlignmentData::MultipleAlignmentData(const MultipleAlignmentDataType &_type,
+                                             const QString &name,
+                                             const DNAAlphabet *_alphabet,
+                                             const QList<MultipleAlignmentRow> &_rows)
+    : type(_type), alphabet(_alphabet), rows(_rows), length(0) {
```

Comment:
You can set a default value to `length` in the .h file and do not set it here

Reply:
Fixed